src/cmake/modules/FindOpenColorIO.cmake

```diff
@@ -45,7 +44,18 @@ MACRO(FindOpenColorIO)
         /sw/lib
         /opt/local/lib
         DOC "The OCIO library")
-
+    option(OCIO_STATIC OFF "Use STATIC lib mean use specific config for use")
```

Comment:
In order to deal with dll import/export OCIO define OpenColorIO_EXPORTS which define macros for import or export but which should be empty in case OCIO is build as static lib. To avoid windows error when building OIIO with static OCIO, I defined OpenColorIO_STATIC preprocessor (pull request : https://github.com/imageworks/OpenColorIO/pull/392).
I just realized that the one for tinyxml should be fixed too... sorry.
Should I correct according to all your comments and commit again (and redo the pull request)? Or it's not necessary? Thanks.